### PR TITLE
docs: add real-time query metrics GUCs to reference

### PR DESCRIPTION
**Pull request by litzell**
Adds gp_enable_query_metrics and gp_instrument_shmem_size GUCs to reference guide

### CODE REVIEW

**Comment by benchristel** [comment]
@litzell I actually don't know that much about the metrics collector. Maybe @violet1986 or @magi345 should review this?

**Comment by litzell** [comment]
@benchristel of course. sorry about that!

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -3431,6 +3437,41 @@
...
+    <title>gp_enable_query_metrics</title>
+    <body>
+      <p>Enables collection of query metrics. When query metrics collection is enabled, 
+        Greenplum Database collects metrics during query execution. The default is off.</p> 
+      <p>After changing this configuration parameter, Greenplum Database must be restarted for the 
+        change to take effect.</p>
+      <p>The Greenplum Database metrics collection extension<sup>1</sup>, when enabled, sends the
+      collected metrics over UDP to a Pivotal Greenplum Command Center<sup>1</sup> agent. </p>
+      <note><sup>1</sup> The metrics collection extension is included in the commercial
+        version of Greenplum Database. Pivotal Greenplum Command Center is supported only with the
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: I think in general we want to talk about Pivotal Greenplum Database specifically. There could be other "commercial" releases of Greenplum Database at some point even though right now it's just Pivotal Greenplum Database.

File: gpdb-doc/dita/ref_guide/config_params/guc_category-list.xml [hunk 2]
```diff
@@ -925,6 +925,31 @@
         </simpletable>
       </body>
     </topic>
+    <topic id="query-metrics">
+      <title>Query Metrics Collection Parameters</title>
+      <body>
+        <p>These parameters enable and configure query metrics collection. When enabled, 
+          Greenplum Database saves metrics to shared memory during query execution. These
+          metrics are used by Greenplum Command Center, which is included with the 
+          commercial version of Pivotal Greenplum Database.</p>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc_category-list.xml]: Maybe it should be "Pivotal's commercial release of Greenplum Database".

**Pull request merged**